Source/BottomView/BottomContainerView.swift

```diff
@@ -62,29 +64,35 @@ open class BottomContainerView: UIView {
 
   // MARK: Initializers
 
-  public override init(frame: CGRect) {
-    super.init(frame: frame)
+  public init(configuration: Configuration?) {
```

Comment:
@onmyway133 what if other initializer is called, e.g. `init(frame: )`? Then, the configuration will not be initialized. So, I've decided to go for optional. 
How about setting default to `nil` ?